aequilibrae/transit/transit.py

```diff
@@ -0,0 +1,52 @@
+import os
+from aequilibrae.log import logger
+from aequilibrae.project.database_connection import database_connection
+from aequilibrae.transit.lib_gtfs import GTFSRouteSystemBuilder
+
+
+class Transit:
+    default_capacities = {
+        0: [150, 300, 300],  # Tram, Streetcar, Light rail
```

Comment:
Do we still have three columns for capacities in transit? Or only two?

Reply:
I believe we've talked about keeping only seated and total capacities. I'll remove design capacity column.